### PR TITLE
chore: Update domain in Backstage catalog

### PR DESCRIPTION
**Pull request by SpotOn-BuildAdmin**
This pull request introduces updates to the Backstage catalog-info.yaml file to ensure the repository's metadata is current and accurate.

### Summary of Changes
- Updated the repository description in catalog-info.yaml.
- Added a workflow to check the correctness of the catalog-info.yaml file.

### Actions Required
1. **Review and Confirm Data**: Please verify the accuracy and completeness of the changes made to the catalog-info.yaml file.
2. **Provide Correct Description**: Ensure that the repository description in the catalog-info.yaml file is accurate and up-to-date.
3. **Repository Archival**: If you determine that this repository is no longer needed, please archive it or contact devops to do it for you.

Your timely review and action on these points will help maintain the integrity and organization of our project repositories.

Thank you!

### CODE REVIEW

File: .spoton/catalog-info.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ metadata:
   annotations:
     github.com/project-slug: spotoninc/renovate-config
   # backstage.io/techdocs-ref: dir:.
+    spoton.com/domain: domain:default/hospitality
```
Comment (MaxymVlasov) [inline .spoton/catalog-info.yaml:12]: I suppose

```suggestion
    spoton.com/domain: domain:default/open-source
```

**Review by MaxymVlasov — APPROVED** [review]

**Comment by JacekPietrasSpotOn** [comment]
@MaxymVlasov there is no such domain like `open-source`

**Comment by MaxymVlasov** [comment]
@JacekPietrasSpotOn would be nice to have a list of domains them, and who part of which domain

**Comment by JacekPietrasSpotOn** [comment]
@MaxymVlasov 
There are sadly multiple sources of truth:

- https://docs.google.com/spreadsheets/d/1dZTg_ckbSZ0EmoM5FNV0vZkwvNwFKTD_5SCrnez1ld4/edit#gid=2099335506
in this document on other sheets there are people 

- you can also check it in charthop:
https://app.charthop.com/spoton/sheet?columns=name%2Cdomain%2Csubdomain&sort=name&total=sum&filter=name%3A%22Maciej%20Marz%C4%99ta%22

- better diagrams are available there:
https://lucid.app/lucidchart/f68922e0-72b3-49f5-8036-04d41b08d380/edit?invitationId=inv_358d5776-bcaa-40cb-af60-09014bd64c09&referringApp=slack&page=oE.mxJ6M4ONv#

**Pull request merged**